Makefile

```diff
@@ -30,5 +36,12 @@ clang-tidy:
 	# DeprecatedOrUnsafeBufferHandling: See https://stackoverflow.com/a/50724865/945780
 	clang-tidy zcfan.c -checks=-clang-analyzer-security.insecureAPI.DeprecatedOrUnsafeBufferHandling -- $(CFLAGS) $(LDFLAGS)
 
+install: all
+	mkdir -p $(DESTDIR)$(bindir)/
+	$(INSTALL) -pt $(DESTDIR)$(bindir)/ $(EXECUTABLES)
+ifeq ($(HAS_LIBSYSTEMD),1)
```

Comment:
More a comment for psi-notify, I suppose, but it's conceivable that someone might not want libsystemd integration but still wants the service file. So maybe just install it unconditionally? :-)

Reply:
Oh, and for this case I guess it might trigger as well if it's built in a chroot without libsystemd as a dependency, but we'd still want it then.

Reply:
that makes sense, I can rework. It will also let me drop a build dependency